bundles/AdminBundle/Resources/public/css/editmode.css

```
@@ -480,6 +480,9 @@ input.cke_dialog_ui_input_tel {
 .pimcore_areablock_toolbar {
     opacity: .9;
     border-color: #000;
+    overflow-y: auto;
+    max-height: 90%;
```

what about
```suggestion
    max-height: calc(100vh - 15px);
```
just to not change from the former values defined in js?

---

Thanks, verified this works and will commit